test/run_test.sh

```diff
@@ -86,7 +90,8 @@ $MOVE_LOG_COMMAND || true
 # Injector commands for re-use
 # We can either test --simulate, and get partial data, or get full data
 # It's better to get full data
-INJECTOR_BASE_CMD="privcount inject --log events.txt"
+GZIP_CMD="gzip -c -d events2.txt.gz"
```

Comment:
Maybe the question I should be asking is:
Why do we need a 3MB file for test data anyway?
It's going to slow down the integration testing significantly.
(Or, more likely, the integration testing will stop before the file is fully processed.)